### ISSUE TITLE
Typescript Error when declaring a jBox tooltip as type 'Mouse'

### ISSUE THREAD

**Issue by fredclausen** [StephanWagner/jBox#172]
Typescript gives the following error

```
Argument of type '"Mouse"' is not assignable to parameter of type 'keyof jBoxOptionsMap'.
```

When compiling a jBox tooltip that is declared like this

```typescript
const freq_tooltip = new jBox("Mouse", {
```

Adding the following to the `jBoxOptionsMap` interface (declared in jBox.d.ts)  appears to rectify the issue

```typescript
declare interface jBoxOptionsMap {
  Tooltip: jBox.jBoxOptions;
  Modal: jBox.jBoxOptions;
  Confirm: jBox.jBoxConfirmOptions;
  Notice: jBox.jBoxNoticeOptions;
  Image: jBox.jBoxImageOptions;
  Mouse: jBox.jBoxOptions;
}
```

But I'm not versed enough in either typescript or the jBox code base to know if that is a valid fix or not.

**Comment by StephanWagner** [comment]
Should be fixed in version v1.3.3

**Comment by fredclausen** [comment]
Thank you!
